src/services/zapper.js

```diff
@@ -37,9 +37,13 @@ type ZapperAsset = {
   tokenAddress: string,
   label?: string,
   img?: string,
+  noImage?: boolean,
```

Comment:
Do you think we need noImage? Could you just `img: ?string` or something similar? However saying this - i'm aware there could be a good reason for the explicit boolean...

Reply:
👍